PendingReleaseNotes

```diff
@@ -148,6 +148,8 @@ CephFS: Disallow delegating preallocated inode ranges to clients. Config
    `ceph config get mgr mgr/volumes/snapshot_clone_no_wait`
   and it can be disabled by using:
    `ceph config set mgr mgr/volumes/snapshot_clone_no_wait false`
+* CephFS: cephfs-journal-tool prevent itself from working on an active filesystem.
```

Comment:
reword this as: cephfs-journal-tool is guarded against running on an online file system.